### DISCUSSION TITLE
Hoping someone can help me create a visual graph of programs

### DISCUSSION THREAD

**Discussion by Tuttle13** [sebr/bhyve-home-assistant#93] [Q&A]
Hi, absolutely loving the addon, has made a "smart" device next-level intelligent!
I'm in Aus, and found the official apps rain prediction and rainfall amount to often be very wrong.

Instead I'm automating it in HA, along with my local weather station, and multiple triggers on hot days etc to auto enable other programs (Orbit really need to add the ability to create more than 3 active programs).

Anyway, my question is, is there someone here who has the ability (and the time/kindness) to help me create a visual representation of my daily watering schedules in HA? Kind of like the calendar view in the official app, but just a daily view, and incorporating both my Bhyve devices (official app makes you select one of them at a time).

I can see the two attributes required, which would be the "start time" and "run time" (in minutes).
I am running 3 programs (so 6 in total) on each of the devices (one is "morning" on is "afternoon" on is "hot day")

I have zero ability or knowledge on how to time bar graph this, having only recently worked out how to use apex graphs for a live power price plot.

Any help and direction would be much appreciated!

Edit: also pretty new to HA in general, so if there is a more suitable place for me to ask this, then please let me know! :)

**Comment by sebr** [comment]
Hey @Tuttle13 , glad the integration is helping you out! I'd suggest for a query like this you'd get a much more helpful responses if you ask in the HA community forums as the question is indeed quite a complex one that stretches multiple different areas of HA. You can find the forums here: https://community.home-assistant.io/
